mlops_proj/src/webapp.py

```
@@ -0,0 +1,69 @@
+        data = np.array([
+            input_data.user_id,
+            input_data.content_id,
+            input_data.watch_seconds,
+            input_data.rating,
+            input_data.popularity
+        ])
+        recommend = inference(model, scaler, label_encoder, data)
+        recommend = [int(r) for r in recommend]
+        return {"recommended_content_id": recommend}
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)